src/style.css

```diff
@@ -1,3 +1,103 @@
+:root {
+  --blue-color: #00aadc;
+}
+
+body,
+header,
+nav,
+h1,
+h2,
+h3,
+p,
+ul,
+li,
+a {
```

Comment:
The use of the universal selector to reset margins and paddings is against the checklist requirements due to potential performance impacts. Consider using a more specific approach to reset these properties.